src/index.js

```diff
@@ -1,5 +1,227 @@
+/**
+ * Studio Meta tailwind-config
+ * 
+ * @package tailwind-config
+ * 
+ * @since 1.0.0-alpha.0
+ * 
+ * Full tailwind default config file
+ * @see https://github.com/tailwindcss/tailwindcss/blob/master/stubs/defaultConfig.stub.js
+ * 
+ */
 module.exports = {
-  theme: {},
-  variants: {},
+  theme: {
+    /**
+     * Breakpoints
+     */
+    screens: {
+      xxs: '0px',
+      xs: '480px',
+      s: '768px',
+      m: '1024px',
+      l: '1280px',
+      xl: '1440px',
+      xxl: '1920px',
+    },
+
+    /**
+     * Colors
+     */
+    colors: {
+      transparent: 'transparent',
+      white: '#fff',
+      black: '#000',
+    },
+    textColor: theme => theme('colors'),
+    backgroundColor: theme => theme('colors'),
+    fill: theme => ({
+      ...theme('colors'),
+      current: 'currentColor'
+    }),
+    borderColor: theme => theme('colors'),
+    placeholderColor: theme => theme('colors'),
+    stroke: theme => ({
+      ...theme('colors'),
+      current: 'currentColor'
+    }),
+
+    /**
+     * Spacing
+     */ 
+    spacing: {
+      auto: 'auto',
+      0: '0',
+      1: '1',
+      2: '2',
+      4: '4',
+      8: '8',
+      16: '16',
+    },
+    margin: (theme, { negative }) => ({
+      auto: 'auto',
+      ...theme('spacing'),
+      ...negative(theme('spacing')),
+    }),
+    padding: theme => theme('spacing'),
+    
+    /**
+     * Flex
+     */
+    flex: {
+      '1': '1 1 0%',
+      auto: '1 1 auto',
+      initial: '0 1 auto',
+      none: 'none',
+    },
+    flexGrow: {
+      '0': '0',
+      default: '1',
+    },
+    flexShrink: {
+      '0': '0',
+      default: '1',
+    },
+    order: {},
+
+    /**
+     * Typography
+     */
+    fontFamily: {},
+    fontSize: {},
+    fontWeight: {
+      hairline: '100',
+      thin: '200',
+      light: '300',
+      normal: '400',
+      medium: '500',
+      semibold: '600',
+      bold: '700',
+      extrabold: '800',
+      black: '900',
+    },
+    letterSpacing: {},
+    lineHeight: {},
+    listStyleType: {},
+    inset: {},
+
+    /** 
+     * Size
+     */
+    container: {},
+    width: theme => ({
+      auto: 'auto',
+      ...theme('spacing'),
+      full: '100%',
+      screen: '100vw',
+    }),
+    maxWidth: (theme, { breakpoints }) => {
```

Comment:
I am not sure the `max-width` utilities will be used. Maybe we could remove them in this first draft?